specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -2605,6 +2605,57 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/querytriggers": {
```

Comment:
Why create this new API? Can't you use the existing `Triggers_ListByFactory` action with query parameters to do this?

Reply:
As explained by hermine in the comments earlier. The pattern that our service is following right now with all the existing APIs is that query is always a post because we do not want filters in the url. Only lists and point gets are a GET call(list would return a nextlink). But for query we want to return a continuation token. Also to fit the current design we have, query calls should be a post action on datafactory